### DISCUSSION TITLE
[Feature Request] have an overview feature for cinnamon

### DISCUSSION THREAD

**Discussion by hazelshantz** [linuxmint/discussions#534] [Ideas]
so far I love cinnamon, but when compared to gnome and plasma, it could take some notes from their overview feature. also showing all desktops and having QoL improvements.

**Comment by spamegg1** [comment]
Do you mean this? You can press Ctrl+Alt+Up arrow, it also lets you drag windows from one to the other, add more than 4, rename them, etc.
![Screenshot from 2024-07-22 00-30-46](https://github.com/user-attachments/assets/afb7681e-63ba-4a0d-83d4-791d31dbd2a8)

**Reply by ghost** [reply]
Also, Ctrl+Alt+Down for a detailed overview on the current workspace

**Comment by JosephMcc** [comment]
Suggestion are always welcome but what you've provided here isn't much use. Things like WHAT could be improved? WHY would that make it better? Give some actual examples and use cases. Just saying "make it better"  really isn't helpful and will result in this basically just being looked over and not even really given much thought.

**Comment by goyalyashpal** [comment]
no replies since 9 months.  warrants closing the discussion.
